lib/python/asterisk/realtime_converter.py

```diff
@@ -50,9 +51,11 @@ class offers a simple way to ensure that the file is restored to its
     """
 
     def __init__(self, config_dir, filename):
-        self.file = os.path.join(config_dir, filename)
+        file_path = config_dir + "/" + filename
```

Comment:
You should probably still use os.path.join instead of concatenating the strings.